draco/cli.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
```

Comment:
I removed the CLI on purpose since I think we should use the Python interfaces instead.

Reply:
The goal here is to support more diverse use cases. CLIs are usually limited to specific applications and don't allow for much flexibility. However, we want Draco to be used for the recommendation of charts given various inputs, enumeration of a space of possible charts, checking specs, etc.

Reply:
sure, will be removing this file then.